client/my-sites/people/people-notices/index.jsx

```diff
@@ -101,6 +101,20 @@ class PeopleNotices extends React.Component {
 					context: 'Error message after A site has failed to perform actions on a user.',
 				} );
 			case 'RECEIVE_DELETE_SITE_USER_FAILURE':
+				if ( 'user_owns_domain_subscription' === log.error.error ) {
+					return i18n.translate(
+						'@%(user)s owns following domain(s) used on this site: {{strong}}%(domains)s{{/strong}}. Domains owned by this user will have to be transferred to a different site, transferred to a different registrar, or canceled before removing or deleting @%(user)s.',
```

Comment:
Late to the party, but it's good to remember that we have support for plural forms in `i18n`: https://github.com/Automattic/i18n-calypso#pluralization. It's a pet-peeve of mine when messages use "stuff(s)", instead of relying on actually counting the number of items and showing an appropriate message ;) Of course, this is a nitpick, so just something to keep in mind in the future.

Reply:
Oh!  TIL. Thanks!